Automattic/jetpack
# Masterbar: rewrite tracks-events.js without jQuery

This PR rewrites `tracks-events.js` without the `jQuery` dependency. This change should be transparent, with no behavioural or functional changes.

Note that since we're talking about a tracking feature, and since it helps to know about old browsers, I opted for attempting to maintain as high a level of compatibility as possible through the use of ES5 and some IE11 workarounds.

#### Changes proposed in this Pull Request:
* Rewrite `tracks-events.js` without jQuery
* Remove `tracks-events.js` from the ESLint exclusion list

#### Jetpack product discussion
None.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:






* Go to `/wp-admin` on your test site
* Open DevTools
* Switch to the Network tab
* Ensure "Preserve log" is enabled on the network tab
* Filter by Fetch/XHR
* Click "Reader" on the masterbar, followed by "Followed sites" on the sidebar that pops open
* Consult the Network tab in DevTools to ensure that a request was made to `admin-ajax.php`, with `action: jetpack_tracks`, `tracksEventName: masterbar_click`, and `tracksEventProp[clicked]: reader_followed_sites` in its payload, among other data.
